tools/Environments/DevHome.Environments/ViewModels/ComputeSystemViewModel.cs

```diff
@@ -41,6 +42,10 @@ public partial class ComputeSystemViewModel : ComputeSystemCardBase, IRecipient<
 
     public ComputeSystemCache ComputeSystem { get; protected set; }
 
+    private readonly object _lock = new();

```

Comment:
nit: this lock is specifically for Properties collection. Properties can be used itself as the locking object.

Reply:
oh thats fair, updated.